masonry/src/widget/text_area.rs

```diff
@@ -738,8 +788,17 @@ impl<const EDITABLE: bool> Widget for TextArea<EDITABLE> {
 
     fn update(&mut self, ctx: &mut UpdateCtx, event: &Update) {
         match event {
-            Update::FocusChanged(_) => {
-                ctx.request_render();
+            Update::FocusChanged(focused) => {
+                // HACK: currently, when moving focus away from a text area, the Ime::Disabled
+                // event is routed to the newly focused widget. Do IME clean up here.
+                if !focused && self.editor.is_composing() {
```

Comment:
As a code style thing, I'd probably write this as an:
```rust
FocusChanged(false) if self.editor.is_composing() => {
```
arm. But I think this code should go anyway, so that's moot.

Reply:
Gone. :wind_face: